lib/inspec/profile.rb

```diff
@@ -206,7 +211,12 @@ def collect_tests(include_list = @controls)
           next if content.nil? || content.empty?
 
           abs_path = source_reader.target.abs_path(path)
-          @runner_context.load_control_file(content, abs_path, nil)
+          begin
+            @runner_context.load_control_file(content, abs_path, nil)
```

Comment:
Thanks for doing this! So I'm guessing the flow here is during the `collect_tests` phase for the profile object, this is where you check and kill things if `load_control_file` fails, to isolate and document that failure against the Profile